sphinx-docs/preprocess.py

```diff
@@ -18,20 +18,14 @@ def replace_in_file(filepath: Path, original_str: str, replace_str: str) -> None
 if __name__ == '__main__':
     sphinx_root = Path(__file__).absolute().parent
     repository_root = sphinx_root.parent
-    markdown_root = sphinx_root / "source" / "md"
+    docs_root = sphinx_root / "source" / "docs"
     repository_url = "https://github.com/microsoft/InnerEye-DeepLearning"
 
-    # Create directories source/md and source/md/docs where files will be copied to
-    if markdown_root.exists():
-        shutil.rmtree(markdown_root)
-    markdown_root.mkdir()
-
-    # copy README.md and doc files
-    shutil.copy(repository_root / "README.md", markdown_root)
-    shutil.copy(repository_root / "CHANGELOG.md", markdown_root)
-    shutil.copytree(repository_root / "docs", markdown_root / "docs")
+    # copy README.md and CHANGELOG.md files
+    shutil.copy(repository_root / "README.md", docs_root)
+    shutil.copy(repository_root / "CHANGELOG.md", docs_root)
```

Comment:
Maybe it's cleaner to create [symlinks](https://docs.python.org/3/library/pathlib.html#pathlib.Path.symlink_to)?

Reply:
Symlinks still require access to files outside of the `sphinx-docs` folder - if sphinx works like docker then this wouldn't work (not sure if that's the case or not with sphinx though).

Reply:
That's fine. See the symlink at https://github.com/microsoft/hi-ml/blob/main/docs/source/multimodal.md